graphql/queryResolvers/moduleCrud.ts

```diff
@@ -0,0 +1,44 @@
+import type { MutationDeleteModuleArgs, MutationAddModuleArgs } from '..'
+import prisma from '../../prisma'
+import { Context } from '../../@types/helpers'
+import { isAdminOrThrow } from '../../helpers/isAdmin'
+import type { Module } from '@prisma/client'
+
+export const modules = (): Promise<Module[]> => {
+  return prisma.module.findMany({
```

Comment:
Do we want to an error to be thrown when findMany throws an error, or we do we want to just return an empty array?

Reply:
> would we rather just return an empty array?

I think this hides the array from the end user, who are aspiring engineers. I think its better for them to realize there's an error so they can report it.

> Do we want to an error to be thrown when findMany throws an error

Graphql will automatically catch and log the error, which gets logged into sentry with line number and the exact error. Here's an example:
<img width="638" alt="Screen Shot 2022-02-13 at 9 41 59 AM" src="https://user-images.githubusercontent.com/686933/153767527-61101c21-7064-4569-a086-f11c84232f1b.png">

So I think we are sufficiently covered

Reply:
Oh wow. So with Sentry, we don't have to do any error logging ourselves?

Reply:
Wow that's awesome! Thanks for the context @songz